pkg/log/audit.go

```diff
@@ -0,0 +1,50 @@
+package log
+
+import (
+	"fmt"
+	"strings"
+
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
+)
+
+const (
+	lineLength = 40
+
+	messageSuccess = "SUCCESS"
+	messageSkipped = "SKIPPED"
+	messageFailed  = "FAILED " // leave the trailing space for consistent lengths
+)
+
+// Audit displays a message to the user. This shouldn't be used for debug logging purposes; all
+// messages passed in here should be user-readable.
+func Audit(message string) {
+	fmt.Println(message)
+}
+
+func AuditComponentSuccessful(component string) {
+	message := formatComponentStatus(component, messageSuccess)
+	Audit(message)
+}
+
+func AuditComponentSkipped(component string) {
+	message := formatComponentStatus(component, messageSkipped)
+	Audit(message)
+}
+
+func AuditComponentFailed(component string) {
+	message := formatComponentStatus(component, messageFailed)
+	Audit(message)
+}
+
+func formatComponentStatus(component, status string) string {
+	// Example output:
+	// Component ... [STATUS]
+
+	name := cases.Title(language.English).String(component)
```

Comment:
A slight inconvenience is that this will not work properly for acronyms e.g. `RPM` is translated to `Rpm` here.

Reply:
I noticed, it kinda drove me a bit nuts. I thought about adding an optional flag to "skip formatting" the component but felt like overkill.